serverless.js

```diff
@@ -202,6 +202,25 @@ module.exports = {
         }}`,
       },
     },
+    resubscribeCalendars: {
+      handler:
```

Comment:
Is 16 seconds of default `timeout` enough for this lambda? We have quite a few orchestrations to do. Maybe we can bump this a bit, just to be safe. What do you think?